.github/workflows/benchmarks.yml

```diff
@@ -3,7 +3,7 @@ name: Benchmarks (CodSpeed)
 on:
   push:
     branches:
-      - main
+      - dev
```

Comment:
@Eh2406 i used the wrong branch name, that's why we currently don't get data from main.